### PR TITLE
Add layout template

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/211
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/732
https://github.com/open-orchestra/open-orchestra-model-interface/pull/288
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/252
https://github.com/open-orchestra/open-orchestra/pull/1143
https://github.com/open-orchestra/open-orchestra-front-demo/pull/333
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2095

### CODE REVIEW

File: BackofficeBundle/Controller/NodeController.php [hunk 1]
```diff
@@ -71,7 +76,7 @@ public function formAction(Request $request, $siteId, $nodeId, $language, $versi
                 }
             }
 
-            $this->get('object_manager')->flush();
+            //$this->get('object_manager')->flush();
```
Comment (ngilain) [inline BackofficeBundle/Controller/NodeController.php]: dead code

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Node/NodeView.js [hunk 2]
```diff
@@ -191,6 +191,8 @@ class NodeView extends OrchestraView
      * @private
      */
     _activeArea(event) {
+        console.log('active');
+        console.log('active');
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Node/NodeView.js]: debug to remove

File: BackofficeBundle/Resources/public/templateSet/default/column_left.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<div class="row">
+    <div class="col-lg-12">
+        <div class="area header disabled">
+            <span>Header</span>
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/templateSet/default/column_left.html]: We should find a way to use translation here : if the BO is turned in french, this won't be turned too

File: BackofficeBundle/Resources/translations/validators.fr.yml [hunk 4]
```diff
@@ -6,6 +6,7 @@ open_orchestra_backoffice_validators:
         pattern: Le bloc %blockLabel% nécessite le paramètre %parameter% dans la pattern de l'url du node
         restore: Vous ne pouvez pas restaurer ce noeud, vous devez restaurer son parent %nodeparent% avant
         unique_node_order: L'emplacement de la page doit être unique
+        unique_node_special_page: Il existe déjà une page pour cette page spéciale
```
Comment (ngilain) [inline BackofficeBundle/Resources/translations/validators.fr.yml]: maybe rather something like "Il existe déjà une page spéciale de ce type"

File: BackofficeBundle/Resources/translations/validators.en.yml [hunk 5]
```diff
@@ -6,6 +6,7 @@ open_orchestra_backoffice_validators:
          pattern: The block %blockLabel% need parameter %parameter% in node pattern url
          restore: You can not restore this node, you should be restore her parent %nodeparent% before
          unique_node_order: The page location must be unique
+         unique_node_special_page: There is already a page for this special page
```
Comment (ngilain) [inline BackofficeBundle/Resources/translations/validators.en.yml]: maybe rather something like "There is aleady a special page of this kind"

**Comment by alavieille** [comment]
@ngilain pr updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**